.storybook/main.ts

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   addons: [
     "@storybook/preset-create-react-app",
-    "@storybook/addon-knobs/register",
+    "@storybook/addon-knobs/register"
```

Comment:
Does `@storybook/addon-docs` need to be added here? It looks like the documentation tab is missing now:

<img width="659" alt="image" src="https://user-images.githubusercontent.com/33756113/97979942-03292b00-1d9e-11eb-900e-8517257faa86.png">

Reply:
Oh; great catch!  Yes.